src/public/modules/forms/admin/directiveViews/edit-form.client.view.html

```diff
@@ -304,6 +329,7 @@
                   <button
                     class="current-field-opt-icon"
                     ng-click="tryDuplicateField($index, isCondition(field) || isHidden(field)); $event.stopPropagation()"
+                    ng-show="field.myInfo && numMyInfoFields < maxMyInfoFields"
```

Comment:
this disallows duplication of non-MyInfo fields. I think this might need to change to the following:
```suggestion
                    ng-show="!field.myInfo || numMyInfoFields < maxMyInfoFields"
```

Reply:
Good catch!